src/elements/content-explorer/GridViewItem.scss

```diff
@@ -0,0 +1,19 @@
+.bdl-GridViewItem-itemThumbnail {
+    margin-bottom: 10px;
+    padding-bottom: 65%;
+    position: relative;
+
+    .bdl-GridViewItem-itemIcon {
+        align-items: center;
+        bottom: 0;
+        display: flex;
+        height: 50%;
```

Comment:
We should avoid using relative dimensions when the parent doesn't have set dimensions. This icon should have a fixed size and be positioned via flexbox.